packages/hoprd/releases.json

```diff
@@ -5,28 +5,39 @@
     "environment_id": "master-goerli", // the environment id to run release nodes in, see protocol-config.json
     "deprecated": false, // if true, this release is kept for historical purposes and not re-deployed
     "version_major": 0, // optional: major semver for the release
-    "version_minor": 0 // optional: minor semver for the release
+    "version_minor": 0, // optional: minor semver for the release
+    "docker_image": "gcr.io/hoprassociation/hoprd:latest" // FQ docker image name
   },
   {
     "id": "master-xdai",
     "git_ref": "refs/heads/master",
     "environment_id": "master-xdai",
-    "deprecated": false
+    "deprecated": false,
+    "docker_image": "gcr.io/hoprassociation/hoprd:latest"
+  },
+  {
+    "id": "integration-test",
+    "git_ref": "refs/heads/integration/release-environment",
+    "environment_id": "master-xdai",
+    "deprecated": false,
+    "docker_image": "gcr.io/hoprassociation/hoprd:latest"
```

Comment:
this cannot be, since the code in latest is from master, not this branch

Reply:
This is just for testing google cloud deployment itself, I'm not yet deploying the correct code via docker.